pyengine/katana.py

```diff
@@ -1,4 +1,21 @@
-def matcher(qty,book):
+def matcher(side,qty,book):
+
+    # buying or selling determines whether we look for the lowest or highest price
+    if side == 'buy':
+        print('order is BUY, will check market data for lowest price')
```

Comment:
print is a little ambiguous, we are checking that p1 is < p2, rather than returning lowest value